src/components/d2l-teacher-course-creation-admin.js

```diff
@@ -67,8 +67,12 @@ class TeacherCourseCreationAdmin extends BaseMixin(LitElement) {
 			.getAssociations()
 			.then(associationsArray => {
 				let i = 0;
-				associationsArray.map(association => association.RowId = i++);
-				this.associations = associationsArray;
+				if (associationsArray) {
+					associationsArray.map(association => association.RowId = i++);
+					this.associations = associationsArray;
+				} else {
+					this.associations = [];
```

Comment:
```suggestion
					this.associations = Array();
```
The lit-analyzer likes to get upset about `[]` for some reason (although it doesn't look like it complained in CI)